source/slang/slang-ir-inline.cpp

```diff
@@ -33,17 +35,18 @@ struct InliningPassBase
     }
 
         /// Consider all the call sites in the module for inliing
-    void considerAllCallSites()
+    bool considerAllCallSites()
```

Comment:
Definitely not needed right now, but it might be better if the change-tracking going on here were more explicit and passed down. We could have a type `ChangeTracker` or similar that can be passed by pointer/reference down the chain, and then it has a `.setChanged()`, `.hasChanged()`, and `.reset()` members.

Certainly I'd argue that a bare `bool` is not the return type we want in the long run. :)